portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.data;
+
+import org.jetbrains.annotations.Nullable;
+
+public class Comment{
+    //personal data
+    final private String firstName, lastName, email, phone;
```

Comment:
Nit: the convention is usually `private final` instead of `final private`.

Reply:
done on bori_dev_2 branch